sfe/pages/unpause-status.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html dir="ltr" lang="en-US">
+    <header>
+        <title>Unpause - Self-Service Frontend</title>
+        {{ template "meta" }}
+    </header>
+    <body>
+    <div>
+    {{ if eq .ShouldUnpause true }}
```

Comment:
This variable name doesn't make sense to me. "If they should unpause, then display text indicating that they have been unpaused"? Maybe this should be named "UnpauseSuccess" or something?